modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLSubscriptionSender.java

```diff
@@ -30,9 +30,6 @@
 public class DLSubscriptionSender
 	extends GroupSubscriptionCheckSubscriptionSender {
 
-	public DLSubscriptionSender() {
```

Comment:
@W2jx This class is a `Serializable`. I remember that `Serializable` should at least have a constructor without parameters. Please double check.